doc/source/glossary.rst

```diff
@@ -71,11 +73,18 @@ Glossary
         :ref:`list command <list-command>` or see the
         :ref:`Plugin Reference <instruments>`.
 
+    Iteration
+        Iteration refers to the number of times that a workload should be run
+        successfully. It is specified as a configuration parameter, either for
+        a specific workload, a specific section, or in general.
```

Comment:
I'm not sure this needs an entry, as `iterations` the configuration point, is already documented in the agenda reference. And "iteration" the concept has been superseded by "job". There may still be some documentation that talks about "iterations" (a holdover from WA2 which didn't really have jobs), however it is that documentation that should be updated.